tests/CatalogTests/Helpers/GalleryDatabaseQueryServiceTests.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Services.Entities;
+using NuGet.Services.Metadata.Catalog.Helpers;
+using Xunit;
+using CatalogConstants = NuGet.Services.Metadata.Catalog.Constants;
+
+namespace CatalogTests.Helpers
+{
+    public class GalleryDatabaseQueryServiceTests
+    {
+        public class TheBuildDB2CatalogSqlQueryMethod
```

Comment:
If we're just asserting each line of the query in a different test, why don't we just have a single test that asserts the exact query?

Reply:
Readability: because the method name clearly states why that line is tested in the test report. Of course this could all be a single test. Should I merge them into one? Or would you agree there's value in these separate assertions? I typically try to assert only one thing per test.